docs/user-guide/concepts/data-model.md

```diff
@@ -7,31 +7,31 @@ description: Describes the data model of GreptimeDB, focusing on time-series tab
 The data model is mainly based on the table model in relational databases while considering the characteristics of metrics, logs, and events data.
 
-All data in GreptimeDB is organized into tables with names. Each data item in a table consists of three types of columns: `Tag`, `Timestamp`, and `Field`.
+All data in GreptimeDB is organized into tables with names. Each data item in a table consists of three semantic types of columns: `Tag`, `Timestamp`, and `Field`.
 
 - Table names are often the same as the indicator names, log source names, or metric names.
-- `Tag` columns store metadata that is commonly queried.
-  The values in `Tag` columns are labels attached to the collected sources,
-  generally used to describe a particular characteristic of these sources.
-  `Tag` columns are indexed, making queries on tags performant.
+- `Tag` columns uniquely identify the time-series.
+  Rows with the same `Tag` values belong to the same time-series.
+  Some TSDBs may also call them labels.
 - `Timestamp` is the root of a metrics, logs, and events database.
   It represents the date and time when the data was generated.
-  Timestamps are indexed, making queries on timestamps performant.
-  A table can only have one timestamp column, which is called time index.
+  A table can only have one column with the `Timestamp` semantic type, which is also called the `time index`.
 - The other columns are `Field` columns.
   Fields contain the data indicators or log contents that are collected.
   These fields are generally numerical values or string values,
-  but may also be other types of data, such as geographic locations.
-  Fields are not indexed by default,
-  and queries on field values scan all data in the table. It can be resource-intensive and underperformant.
- However, the string field can turn on the [full-text index](/user-guide/logs/query-logs.md#full-text-index-for-accelerated-search) to speed up queries such as log searching.
+  but may also be other types of data, such as geographic locations or timestamps.
 
-### Metric Table
+A table clusters rows of the same time-series and sorts rows of the same time-series by `Timestamp`.
+The table can also deduplicate rows with the same `Tag` and `Timestamp` values, depending on the requirements of the application.
+GreptimeDB stores and processes data by time-series.
+Choosing the right schema is crucial for efficient data storage and retrieval; please refer to the [schema design guide](/docs/user-guide/administration/design-table.md) for more details.
 
-Suppose we have a time-series table called `system_metrics` that monitors the resource usage of a standalone device:
+### Metrics
+
+Suppose we have a table called `system_metrics` that monitors the resource usage of machines in data centers:
 
 ```sql
 CREATE TABLE IF NOT EXISTS system_metrics (
```

Comment:
Is this SQL statement still suitable for the new data model where the tag columns are no longer recommended?

Reply:
Tags are still helpful for querying if we set them properly. We can only use host for tag. But the problem is that I can't modify the SVG......